private/buf/cmd/buf/command/generate/generate.go

```diff
@@ -193,7 +193,6 @@ type flags struct {
 	IncludeWKT      bool
 	ExcludePaths    []string
 	DisableSymlinks bool
-	Types           []string
```

Comment:
Would we not want to also provide a command-line flag, that can override what's in the config file?

Reply:
Maybe some literal interpretation of @bufdev comments above on my part
> We have plans to move more into the buf.gen.yaml into the future

"move" I guess could also mean "include"

Reply:
resolved in [731d00f](https://github.com/bufbuild/buf/pull/1628/commits/731d00f7abd8ddeb8d28c774669ac4ea6fa92d2d)